gui/default/index.html

```diff
@@ -358,38 +358,12 @@ <h4 class="panel-title">
                     <span ng-if="folder.type == 'receiveonly'" class="fas fa-fw fa-download"></span>
                     <span ng-if="folder.type == 'receiveencrypted'" class="fas fa-fw fa-lock"></span>
                   </div>
-                  <div class="panel-status pull-right text-{{folderClass(folder)}}" ng-switch="folderStatus(folder)">
-                    <span ng-switch-when="paused"><span class="hidden-xs" translate>Paused</span><span class="visible-xs" aria-label="{{'Paused' | translate}}"><i class="fas fa-fw fa-pause"></i></span></span>
-                    <span ng-switch-when="unknown"><span class="hidden-xs" translate>Unknown</span><span class="visible-xs" aria-label="{{'Unknown' | translate}}"><i class="fas fa-fw fa-question-circle"></i></span></span>
-                    <span ng-switch-when="unshared"><span class="hidden-xs" translate>Unshared</span><span class="visible-xs" aria-label="{{'Unshared' | translate}}"><i class="fas fa-fw fa-unlink"></i></span></span>
-                    <span ng-switch-when="scan-waiting"><span class="hidden-xs" translate>Waiting to Scan</span><span class="visible-xs" aria-label="{{'Waiting to Scan' | translate}}"><i class="fas fa-fw fa-hourglass-half"></i></span></span>
-                    <span ng-switch-when="cleaning"><span class="hidden-xs" translate>Cleaning Versions</span><span class="visible-xs" aria-label="{{'Cleaning Versions' | translate}}"><i class="fas fa-fw fa-recycle"></i></span></span>
-                    <span ng-switch-when="clean-waiting"><span class="hidden-xs" translate>Waiting to Clean</span><span class="visible-xs" aria-label="{{'Waiting to Clean' | translate}}"><i class="fas fa-fw fa-hourglass-half"></i></span></span>
-                    <span ng-switch-when="stopped"><span class="hidden-xs" translate>Stopped</span><span class="visible-xs" aria-label="{{'Stopped' | translate}}"><i class="fas fa-fw fa-stop"></i></span></span>
-                    <span ng-switch-when="scanning">
-                      <span class="hidden-xs" translate>Scanning</span>
-                      <span class="hidden-xs" ng-if="scanPercentage(folder.id) != undefined">
-                        ({{scanPercentage(folder.id) | percent}})
-                      </span>
-                      <span class="visible-xs" aria-label="{{'Scanning' | translate}}"><i class="fas fa-fw fa-search"></i></span>
+                  <div class="panel-status pull-right text-{{folderClass(folder)}}">
+                    <span class="hidden-xs">{{folderStatusText(folder)}}</span>
+                    <span ng-switch="folderStatus(folder)">
+                      <span ng-switch-when="syncing">({{syncPercentage(folder.id) | percent}}, {{model[folder.id].needBytes | binary}}B)</span>
```

Comment:
Maybe move the `ng-switch` to the `<div>` above, to have one less level of nested `<span>`s.  That should help avoiding stray whitespace between them.

Reply:
>The same applies to scanning, of course. IIUC the intended behavior is this:
>- In the header: Scanning and syncing shows the percentage after the status text on large screens, and without the status text on small screens.
>- In the newly added table row: Any state just shows the status text, we don't need to repeat the percentage there. The icon always reflects the state icon shown in the header, except for scanning and syncing, where the icon is only visible here.

One problem with this is that I think without any icons, it can be confusing whether the folder is currently scanning or syncing. Although the former displays only percentage, and the latter percentage and size, I don't think this is clear to those who aren't already familiar with the internals.

How about just displaying both the icon, percentage, and size together? The icon doesn't take that much space...

Reply:
Yes, we could display the icon in addition.  But frankly I don't think anyone cares what process is currently running, just that it's chugging along and how much is finished.  At least in the overview - if you want more detail, there will be the added row with complete status.